handler/src/main/java/io/netty/handler/ssl/Protocols.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import io.netty.util.internal.ObjectUtil;
+
+import java.util.Arrays;
+
+/**
+ * TLS protocols
+ */
+public final class Protocols {
+
+    /**
+     * TLS v1.0
+     */
+    public static final String TLS_v1_0 = "TLSv1.0";
+
+    /**
+     * TLS v1.1
+     */
+    public static final String TLS_v1_1 = "TLSv1.1";
+
+    /**
+     * TLS v1.2
+     */
+    public static final String TLS_v1_2 = "TLSv1.2";
+
+    /**
+     * TLS v1.3
+     */
+    public static final String TLS_v1_3 = "TLSv1.3";
+
+    public static Iterable<String> of(String... protocols) {
```

Comment:
Also I think we should just remove this method as it not really provide anything useful